enterprisemanagementv1/enterprise_management_v1.go

```diff
@@ -43,7 +43,7 @@ type EnterpriseManagementV1 struct {
 }
 
 // DefaultServiceURL is the default URL to make service requests to.
-const DefaultServiceURL = "https://enterprise.cloud.ibm.com/v1"
+const DefaultServiceURL = "https://enterprise.test.cloud.ibm.com/v1"
```

Comment:
This is incorrect.   The default service URL for the service should be the production endpoint, not the staging endpoing.   It looks like you used the staging/test version of the API definition when generating the SDK code.   You should use the production version of the API instead.